### PR TITLE
Remove rubygems version check from Indexer

### PR DESCRIPTION
**Pull request by sonalkr132**
We are using 2.6.4 now.

### CODE REVIEW

**Comment by dwradcliffe** [comment]
Can you update the readme/contributing docs to say what version is required

**Comment by sonalkr132** [comment]
done :triangular_flag_on_post:

This depends on #1283 btw.

File: app/jobs/indexer.rb [hunk 1]
```diff
@@ -82,25 +82,8 @@ def log(message)
...
-    def self.sanitize_spec(spec)
-      indexer.abbreviate spec
-      indexer.sanitize spec
-    end
-  else # >= 2.5.0
-    def self.sanitize_spec(spec)
-      spec.abbreviate
-      spec.sanitize
-    end
+  def self.sanitize_spec(spec)
```
Comment (arthurnn) [inline app/jobs/indexer.rb]: you can probably remove this method now.

Comment (sonalkr132) [inline app/jobs/indexer.rb]: done :purple_heart:

**Comment by arthurnn** [comment]
Thanks

**Pull request merged**